pkgs/development/python-modules/runstats/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  setuptools,
+  cython,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pytest-cov-stub,
+  pytest-xdist
+}:
+
+buildPythonPackage rec {
+  pname = "runstats";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "grantjenks";
+    repo = "python-runstats";
+    tag = "v${version}";
+    hash = "sha256-YF6S5w/ccWM08nl9inWGbaLKJ8/ivW6c7A9Ny20fldU=";
+  };
+
+  build-system = [
+    setuptools
+    cython
+  ];
+
+  nativeCheckInputs = [
+    pytest-cov-stub
+    pytest-xdist
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "runstats" ];
+
+  meta = {
+    description = "Python module for computing statistics and regression in a single pass.";
+    homepage = "https://github.com/grantjenks/python-runstats";
+    changelog = "https://github.com/grantjenks/python-runstats/blob/${src.rev}/CHANGELOG";
```

Comment:
Is this generated link working?

Reply:
no sorry. i dont know how it got there. i will delete it immediately.

Reply:
No need to delete. You can use tag instead of rev instead.

Reply:
Sorry i just deleted it and now i am looking but there isn't even a CHANGELOG file in the repo. I must have confused it with some other package from my derivations.